packages/next/src/build/templates/pages-api.ts

```diff
@@ -1,14 +1,9 @@
-// @ts-ignore this need to be imported from next/dist to be external
-import * as module from 'next/dist/server/future/route-modules/pages-api/module.compiled'
-
+import { PagesAPIRouteModule } from '../../server/future/route-modules/pages-api/module.compiled'
```

Comment:
Relative imports are possible because the `.js` file references the compiled dist runtime, thereby ensuring it remains external